test/dependencies_for_poisson_solvers.jl

```diff
@@ -17,20 +18,22 @@ function random_divergent_source_term(grid)
     v_bcs = regularize_field_boundary_conditions(default_bcs, grid, :v)
     w_bcs = regularize_field_boundary_conditions(default_bcs, grid, :w)
 
-    Ru = CenterField(grid, boundary_conditions=u_bcs)
-    Rv = CenterField(grid, boundary_conditions=v_bcs)
-    Rw = CenterField(grid, boundary_conditions=w_bcs)
+    Ru, Rv, Rw = VelocityFields(grid, (; u = u_bcs, v = v_bcs, w = w_bcs))
```

Comment:
Since the position of the velocity nodes is hard written here:
https://github.com/CliMA/Oceananigans.jl/blob/ddf322f7faa47980bfe4de5bbd203696f419f9e2/src/Models/NonhydrostaticModels/pressure_correction.jl#L31-L37

this test was not behaving correctly before but since the divergence calculated here is incorrect in the same way it still passed.

I had to change this as we need to add the boundary normal fill, which then failed as it doesn't work on `Center, Center, Center` fields.